Package.swift

```diff
@@ -12,12 +12,19 @@ let package = Package(
     .library(
       name: "DurationPicker",
       targets: ["DurationPicker"]),
+    .library(
+      name: "DurationPickerSwiftUI",
+      targets: ["DurationPickerSwiftUI"]),
   ],
   dependencies: [],
   targets: [
     .target(
       name: "DurationPicker",
       dependencies: []),
+    .target(
+      name: "DurationPickerSwiftUI",
+      dependencies: ["DurationPicker"]
+    ),
```

Comment:
Sooo I don't think this setup works as you intended it to. If I have a pure SwiftUI project, if I add the `DurationPickerSwiftUI` library, and import `DurationPickerSwiftUI` in my class, I still have access to the UIKit version since it's imported transitively.

What you want here is to actually have two _separate_ packages, but that gets a little complicated when hosted in the same repo. I think we can just drop these changes and use the package `DurationPicker` for both to keep things simple